templates/zerver/left-sidebar.html

```diff
@@ -58,7 +58,7 @@
                           </div>
                           <div class="invite-count-area">
                             <span id="invite-hearts"></span>
-                            <small class="pull-right"><span class="invite-count">0</span> {{ _('invite') }}<span class="invite-count-is-plural">s</span> {{ _('remaining') }}</small>
+                            <small class="pull-right"><span class="invite-count">0 {{ _('invites remaining') }}</span></small>
```

Comment:
Could you also include `0` into translate target message? It's useful for translating.